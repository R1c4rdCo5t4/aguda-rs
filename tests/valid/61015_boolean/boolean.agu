-- Author: 61015, Filipe Lopes

let printAndReturn (msg, b) : (String, Bool) -> Bool =
  print(msg);
  b;
  
if false && printAndReturn("This should not print", true) then
  print("Result is true")
else
  print("Result is false");