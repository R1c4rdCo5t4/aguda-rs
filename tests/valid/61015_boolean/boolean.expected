Expected output: Result is false